widget/entry_test.go

```diff
@@ -459,6 +473,12 @@ func TestEntry_ExpandSelectionForDoubleTap(t *testing.T) {
 	start, end = getTextWhitespaceRegion(str, 30)
 	assert.Equal(t, 29, start)
 	assert.Equal(t, len(str), end)
+
```

Comment:
This feels like a new test rather than another case for the one above.
A good name could make it clearer that it's selecting symbols that are considered word delimiters?